klippy/extras/resonance_tester.py

```diff
@@ -198,6 +211,28 @@ def _run_test(self, gcmd, axes, helper, raw_name_suffix=None):
     def cmd_TEST_RESONANCES(self, gcmd):
         # Parse parameters
         axis = _parse_axis(gcmd, gcmd.get("AXIS").lower())
+        accel_chips = gcmd.get("CHIP", None)
+        test_point = gcmd.get("POINT", None)
+
+        if test_point:
+            test_coords = test_point.split(',')
+            if len(test_coords) != 3:
+                raise gcmd.error("Invalid POINT parameter, must be 'x,y,z'")
+            try:
+                test_point = [float(p.strip()) for p in test_coords]
+            except ValueError:
+                raise gcmd.error("Invalid POINT parameter, must be 'x,y,z'"
+                " where x, y and z are valid floating point numbers")
+
+        if accel_chips:
+            initialized_chips = []
```

Comment:
Maybe `parsed_chips` or something like that. 'Initialized' implies that some sort of initialization was performed, but it is not the case.

Reply:
fixed 👍